### ISSUE TITLE
Microsoft Exchange Imap Access

### ISSUE THREAD

**Issue by ychoucha** [RainLoop/rainloop-webmail#887]
Hello,

To acces to my microsoft exchange account i have acces with :+1: 

i have a domain like this : 
mydomain.fr

to acces a imap acces i have to longi like this : 

User : domain\user
password : password

so in rainloop i have to make : 

User : user@mydomain.fr
password : password

i also tried : 

User : mydomain\user@mydomain.fr
password : password

but that's not working, how to make works the rainloop  acces, and where are the log of the application please

**Comment by ychoucha** [comment]
can we acces to exchange with rainloop thanks ?

**Comment by ychoucha** [comment]
Hello, 

can i use rainloop, with mail from an EXCHANGE 2010 ?

thanks

**Comment by wrexem** [comment]
I used the plugin "custom login mapping" to handle this issue: firstname.lastname@domain.com:firstname.lastname and then toggled "use short login name" for the exchange imap domains.

caveat: requires setup for each user

I was actually able to get it to work without this plugin, but my domain's email addresses do not match login names, so I was doing "firstname.lastname@domain.com" for login, but it should've been Firstname.L@domain.com - which I rewrote via that plugin.

Good luck!
